main.go

```diff
@@ -300,7 +301,11 @@ func CreateExecutionPlan(cfg *Config) (Plan, error) {
 					}
 				}
 
-				envStr := "^:|:^" + strings.Join(e, ":|:")
+				envsDelimiter := ":|:"
```

Comment:
Can you make this a const called `defaultEnvVarDelimiter` or similar and move up to the block near line 55?